core/embed/rust/src/micropython/obj.rs

```diff
@@ -415,4 +415,8 @@ impl Obj {
     pub fn is_type_bytes(self) -> bool {
         unsafe { ffi::mp_type_bytes.is_type_of(self) }
     }
+
+    pub fn is_type_str(self) -> bool {
```

Comment:
let's change this to `is_str` and include the `is_qstr` check

Reply:
fixed a0c5483d6e986ee01bf895c0e4abf8ab4710a941